### ISSUE TITLE
Jim McGowan's Book creator issues

### ISSUE THREAD

**Issue by jim-mcgowan** [OpenTechStrategies/SimpleBook#42]
_Note that I am using Book creator in Safari (on a Mac). Should I find different behaviors in other browsers, I will note them accordingly._

- [ ] Should a user leave pages in Book creator when disabling, starting Book creator again triggers the dialog below. Clicking **OK** bypasses the _Start book creator_ page and takes the user directly to the _Manage your book_ page. The user's _old_ pages remain in the book. Clicking **Cancel** leaves the user at the _Start book creator_ page. Clicking **Start book creator** returns them to the original page with 0 pages in the book.

<img width="1440" alt="Click  OK  to continue with your book" src="https://user-images.githubusercontent.com/4147733/119852434-e8064000-bed4-11eb-9b70-b9d29a379dcc.png">

- [x] When Chapters are added to a book, attempts to download the book fail (bringing the user to a  _Book rendering failed_ page).

- [x] Titles and subtitles don’t appear in the rendered PDF.

- [x]  _Letter_ should be the default Paper size.

- [ ] The generated PDF is simply titled book.pdf (rather than the entered book Title).

- [ ] Proposal titles print on their own page rather than as headings with the rest of the proposal’s content.

- [ ] For a single proposal, a user can’t add the Full, LFCAnalysis and Evaluations views to the same book as each view is technically of the same page. Once the Full view is added, for example, a user can’t add the LFC Analysis view as the page is _already added_ to the book.

- [ ] On the main Book creator page, the **Download as PDF** button is misleading. After the book is created, the user is taken to a _Rendering finished_ page with a link to  **Download the file** to your computer. Still the PDF isn’t technically downloaded. This requires an additional step of hovering over the displayed book to find the download button and clicking it.

**Comment by jim-mcgowan** [comment]
Feel free to retitle this issue as desired.

**Comment by slifty** [comment]
I'm making issues for most of these:

Chapters: #37 
Titles / subtitles: #39 
Letter: #40 

* For proposal titles being on their own page we're going to need to support that in the competition templates themselves SimpleBook can't insert elements into the HTML before rendering, but it *can* insert pages into the PDF which is why we did it that way.

* For the evaluations issue I think that's going to be a bigger issue -- it will require per-page view support (we currently have support to define the view for the full book).  I'm going to create that issue!  Once I knock out the others I'll re-evaluate the scope in case there's a faster / clever way to accomplish the goal.

**Comment by slifty** [comment]
A correction -- @frankduncan diagnosed the evaluations issue, @jim-mcgowan is it possible you're using the little drop down menu on the left hand side to switch view, instead of clicking on the tabs?

The tabs each have their own URL and are the way we intend users to navigate.  The drop down UX is currently a bit unintuitive which will warrant its own issue in torque-sites.

**Comment by jim-mcgowan** [comment]
@slifty Yes, I was using the little drop-down. While the drop-down may seem unintuitive, it's going to be more visible to some (like me) than the tabs. Maybe it should be hidden?

**Comment by slifty** [comment]
@jim-mcgowan yes!  That's not a simplebook issue but is a larger issue related to the dropdown (I owe Frank an opened issue for that, based on our group conversation two weeks ago)
